scripts/build.py

```diff
@@ -538,6 +542,11 @@ def build_third_party_libs(third_party_directory_path):
         _generate_copy_tasks_for_fonts(
             dependency_filepaths['fonts'], FONTS_DIR))
 
+    ensure_directory_exists(WEBFONTS_DIR)
```

Comment:
It is confusing that both these dirs (fonts and webfonts) are being copied to dependency_filepaths['fonts']. Can you elaborate more via comments on what's going on here? Are we planning to get rid of the FONTS_DIR in the future (and if so, is there an issue filed for that that clearly describes what remaining work is needed)?

Reply:
Hi @seanlip, Yes we will be removing the `fonts` directory in the next milestone.

Reply:
OK, file an issue for this please and link to it here.

Don't just write "Remove the fonts directory" btw -- the issue description should clearly describe all the different steps that will be done and how we will test that the removal is safe. Write it so that a hypothetical new contributor can take it up if possible. Thanks!

Reply:
Hi @seanlip created an issue #6988 Thanks!